SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/SpatialWorkerConnection.h

```diff
@@ -84,4 +85,6 @@ class SPATIALGDK_API USpatialWorkerConnection : public UObject, public FRunnable
 
 	// RequestIds per worker connection start at 0 and incrementally go up each command sent.
 	Worker_RequestId NextRequestId = 0;
+
+	TOptional<WorkerConnectionCoordinator> ThreadWaitCondition;
```

Comment:
Comment here would be good

Reply:
Why is this an optional? Shouldn't it just be TUnique?

Reply:
It could be, semantically the same but no need to allocate memory (the class itself is mainly just code)

Reply:
Optional uses memory allocation also though right?

Reply:
Nah it stores a small buffer which is allocates into (uint8[sizeof(WorkerConnectionCoordinator)])

Reply:
The schema_option type allocates as the schema options are more like lists with up to one element. Other option types don't.